ale/drivers/__init__.py

```diff
@@ -55,22 +55,37 @@ def default(self, obj):
 
 def load(label, props={}, formatter='ale', verbose=False):
     """
-    Attempt to load a given label from all possible drivers
+    Attempt to load a given label from all possible drivers.
+
+    This function opens up the label file and attempts to produce an ISD in the
+    format specified using the supplied properties. Drivers are used sequentially
```

Comment:
I think I'd say 'Drivers are tried sequentially' rather than 'Drivers are used sequentially' but this might just sound better to (only) me.